### PR TITLE
[ADD] account_payment_cancel

### PR DESCRIPTION
**Pull request by rlizana**
This module allow cancel payment orders in `done` state if lines don't have account move reconciled.

### CODE REVIEW

File: account_payment_cancel/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+{
+    'name': 'Payment Cancel',
+    'category': 'Accounting & Finance',
+    'summary': 'Allow cancel payment order',
+    'version': '0.1',
+    'description': '''
+    ''',
```
Comment (sbidoul) [inline account_payment_cancel/__openerp__.py]: remove description field

File: account_payment_cancel/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    'author': 'Trey Kilobytes de Soluciones (www.trey.es)',
```
Comment (sbidoul) [inline account_payment_cancel/__openerp__.py]: Add Odoo Community Association (OCA) as author.

File: account_payment_cancel/README.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+There is nothing to configure.
+
+Usage
+=====
+
+You are able to add a payment mode directly on a partner.
+This payment mode is automatically associated to the purchase order, then on related invoice.
+This default value could be change in a draft purchase or draft invoice.
+When you create an payment order, only invoices related to chosen payment mode are displayed.
+Invoices without any payment mode are displayed to.
```
Comment (sbidoul) [inline account_payment_cancel/README.rst]: This usage seems unrelated to the module description?

**Comment by rlizana** [comment]
@sbidoul thanks for you comments, I fix it

File: account_payment_cancel/README.rst [hunk 4]
```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Account Payment Purchase
+========================
+
+This module allow cancel payment orders in done state if lines don't have account move reconciled.
```
Comment (pedrobaeza) [inline account_payment_cancel/README.rst]: s/allow/allows

File: account_payment_cancel/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+#
+#    Trey, Kilobytes de Soluciones
+#    Copyright (C) 2014-Today Trey, Kilobytes de Soluciones <www.trey.es>
```
Comment (pedrobaeza) [inline account_payment_cancel/__openerp__.py]: Replace Today to 2015, because today is relative

File: account_payment_cancel/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+{
+    'name': 'Payment Cancel',
+    'category': 'Accounting & Finance',
+    'summary': 'Allow cancel payment order',
+    'version': '0.1',
+    'license': 'AGPL-3',
+    'author': 'Trey Kilobytes de Soluciones - Roberto Lizana (www.trey.es), '
```
Comment (pedrobaeza) [inline account_payment_cancel/__openerp__.py]: You should put only the company, not the web or your name. Put website key for your website URL.

**Comment by pedrobaeza** [comment]
Thanks for the module, Roberto. I'm seeing that this module is very limited, because when you use _account_banking_payment_transfer_, no payment order will be able to be cancelled. The idea when we talked about this on the spanish code sprint was that cancel function triggers the workflow change and a method that unreconciles and remove the generated account entries. This makes this module to depend directly on the previously commented one.

**Comment by rlizana** [comment]
@pedrobaeza effectively payment order will be not able to be cancelled if account_banking_payment_transfer module is installed, I reviewed the module (I've done a PR #198) with the intention of putting the functionality for states `sent` and `rejected` in the same module (`account_banking_payment_transfer`), and thus make it compatible with this PR

**Comment by pedrobaeza** [comment]
Any advance in this?

**Comment by pedrobaeza** [comment]
@rlizana, are you going to continue with this PR?
